config.ini

```diff
@@ -0,0 +1,21 @@
+[irc1]
+username=Testbot
+password=
+host=irc.0x00sec.org
+port=6697
+ssl=True
+channels=#bots
+
+[irc2]
+username=Testbot
+password=
+host=irc.sithmail.com
+port=6697
+ssl=True
+channels=#bots
+
+[slack]
+token=xoxb-224517351521-2BU5lv2jyn1xVMsS541FRtXm
```

Comment:
Are these public tokens?

Reply:
Revoked

Reply:
Make sure you get the one beneath, too, for c2c-slack.